src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -1087,9 +1086,9 @@ public struct S
 ";
             var comp = CreateCompilation(csharp);
             comp.VerifyDiagnostics(
-                // (5,32): error CS0106: The modifier 'readonly' is not valid for this item
+                // (5,32): error CS8657: Static member 'S.M()' cannot be 'readonly'.
```

Comment:
> error CS8657: Static member 'S.M()' cannot be 'readonly'. [](start = 26, length = 58)

Could you also test the symbol API (unless that's covered elsewhere), to illustrate that static members always answer `false` when asked if they are `readonly`?

Reply:
I think this file is a good place to add that.

---
In reply to: [267987488](https://github.com/dotnet/roslyn/pull/34338#discussion_r267987488) [](ancestors = 267987488)